demisto_sdk/commands/init/tests/contribution_converter_test.py

```diff
@@ -105,7 +106,7 @@ def test_convert_contribution_zip_updated_pack(get_content_path_mock, get_python
     integration_description = sample_integration_path / 'Sample_description.md'
     integration_image = sample_integration_path / 'Sample_image.png'
     integration_readme_md = sample_integration_path / 'README.md'
-    unified_yml = integrations_path / 'integration-integration0.yml'
+    unified_yml = sample_integration_path / 'integration-integration0.yml'
```

Comment:
Please also check that the unified YAML file doesn't exist from where it was moved (AKA in the `Integrations` directory).